front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -498,16 +502,43 @@ export default function AssistantNew({
 }
 
 export function LimitReachedPopup({
-  planLimitReached,
+  isOpened,
+  onClose,
+  subscription,
   workspaceId,
 }: {
-  planLimitReached: boolean;
+  isOpened: boolean;
+  onClose: () => void;
+  subscription: SubscriptionType;
   workspaceId: string;
 }) {
   const router = useRouter();
+
+  if (isTrial(subscription)) {
+    return (
```

Comment:
Why a Dialog vs a Popup ooc? I really liked that small red-ish popup I'm sad to see it go for that code path?

Reply:
Thread is [here](https://dust4ai.slack.com/archives/C050SM8NSPK/p1710779762670239).